README.md

```diff
@@ -149,6 +150,20 @@ Adds a class to any tooltip:
 {{ember-tooltip class='tooltip-warning'}}
 ```
 
+#### Container
+
+| Type | HTMLElement \| String \| false |
```

Comment:
I think these backslashes are messing with the formatting of the headers in the rendered markdown

Reply:
Oooooooo hah no, I was just being bad at reading. That makes sense. Making them appear as code blocks would be more clear though.